Botusanu Darius-Gabriel/LAB3,pr3.1/main.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+int main()
+{
+    double x,y,z;
+    printf("Input 2 real numbers: ");
+    scanf("%lf %lf", &x, &y);
+    double pow(double, double);
```

Comment:
why do you reference the prototype of the function here? Is it really needed?
if you get a warning for not referencing it (you shouldn't, as you included `math.h`, then reference it globally, outside the main function